Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -85,6 +85,65 @@ Validate that the Cilium Local Redirect Policy CRD has been registered.
 	   [...]
 	   ciliumlocalredirectpolicies.cilium.io              2020-08-24T05:31:47Z
 
+.. note::
+
+    Local Redirect Policy supports either the socket-level loadbalancer or the tc loadbalancer.
+    The configuration depends on your specific use case and the type of service handling required.
+    Below are the Helm setups to work with ``localRedirectPolicy=true``:
+
+    1. Enable full kube-proxy replacement:
+
+      - This setup is for users who want to replace kube-proxy with Cilium's eBPF implementation
+        and leverage Local Redirect Policy.
+
+      .. parsed-literal::
+
+        helm install cilium |CHART_RELEASE| \\
+            --namespace kube-system \\
+            --set kubeProxyReplacement=true \\
+            --set localRedirectPolicy=true
```

Comment:
I believe the goal here is to give users Helm values for specific cases. Doing it via `helm install` command seems unnecessary (and may be confusing, e.g. regarding types), I would just write values as yaml, e.g.:
```
.. code-block:: yaml

   kubeProxyReplacement: true
   localRedirectPolicy: true
```

Same for other cases.